### PR TITLE
now able to specify rules on cli

### PR DESCRIPTION
**Pull request by ddonahue007**
Example of one rule: (remove-unused-imports)
Screen Shot 2020-06-03 at 11 34 24 AM

Example of two rules: (remove-unused-imports background-image-src-enum)
Screen Shot 2020-06-03 at 11 37 39 AM

No rules:
Screen Shot 2020-06-03 at 11 38 47 AM

### CODE REVIEW

File: packages/eslint-plugin-pf-codemods/index.js [hunk 1]
```diff
@@ -1,47 +1,59 @@
...
-  "form-fix-isValid": require('./lib/rules/form-fix-isValid'),
-  "expandable-rename-expandablesection": require('./lib/rules/expandable-rename-expandablesection'),
-  "rename-toolbar-components": require('./lib/rules/rename-toolbar-components'),
-  "page-header-prop-rename": require('./lib/rules/page-header-prop-rename'),
-  "page-header-move-avatar": require('./lib/rules/page-header-move-avatar'),
-};
+let rules = {};
+let arguments = process.argv;
+
+if (arguments.includes('--rules')) {
```
Comment (redallen) [inline packages/eslint-plugin-pf-codemods/index.js:4]: Can we include this login in the pf-codemods package instead? I don't think the eslint plugin should care what CLI options were passed.

File: packages/pf-codemods/index.js [hunk 2]
```diff
@@ -43,8 +44,15 @@ function printResults(engine, results, format) {
 
 function main() {
   let currentOptions;
+
   try {
-    currentOptions = options.parse(process.argv);
+    let arguments = lodash.cloneDeep(process.argv);
```
Comment (redallen) [inline packages/pf-codemods/index.js:49]: I think `[...process.argv]` works? No need for lodash

File: packages/pf-codemods/index.js [hunk 3]
```diff
@@ -43,8 +44,15 @@ function printResults(engine, results, format) {
...
+
+    // remove --rules for parsing purposes
+    if (process.argv.includes('--rules')) {
+      arguments.splice(arguments.indexOf('--rules'),2);
+    }
+    currentOptions = options.parse(arguments);
   } catch (error) {
     console.error(error.message);
     return 2;
```
Comment (redallen) [inline packages/pf-codemods/index.js:58]: Line 62-83 is what needs editing. `rules: undefined` needs to be populated and the `.eslintrc.json` might need to be modified to not use the recommended rules from the plugin (aka ALL rules).

File: packages/pf-codemods/index.js [hunk 4]
```diff
@@ -2,6 +2,7 @@
 const path = require('path');
 const options = require('eslint/lib/options');
 const { CLIEngine } = require('eslint/lib/cli-engine');
+const lodash = require('lodash');
```
Comment (dlabrecq) [inline packages/pf-codemods/index.js:5]: It's possible `lodash` was installed via another package (e.g., Victory or react-charts), but should add `lodash` as a `pf-codemods` dependency.

File: packages/pf-codemods/index.js (hunk 2 above)
Comment (ddonahue007) [reply packages/pf-codemods/index.js:49]: I was trying to be safe with using lodash.deepcopy(), but I am sure it is over kill for this. I have changed back to the above suggestion.

File: packages/pf-codemods/index.js (hunk 4 above)
Comment (ddonahue007) [reply packages/pf-codemods/index.js:5]: I removed the need to use lodash as it is most likely over kill for our needs here.

File: packages/pf-codemods/index.js (hunk 3 above)
Comment (ddonahue007) [reply packages/pf-codemods/index.js:57]: Done

**Pull request merged**